app/assets/javascripts/tasks.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```

Comment:
Feel free to delete empty/unused files like this one that Rails generates. I used to forget to do this, but it's a good habit pick up :)